client/src/store/utils.ts

```diff
@@ -0,0 +1,80 @@
+import { CreateApiOptions } from '@reduxjs/toolkit/query/react';
```

Comment:
These are the factory functions that I factored out of store/admin/api, but decided to keep them factored out even though I didn't end up reusing them in the register store.  I think it makes store/admin/api a little easier to grok